code/modules/mining/mine_turfs.dm

```diff
@@ -10,7 +10,7 @@ var/global/list/rockTurfEdgeCache
 	name = "rock"
 	icon = 'icons/turf/mining.dmi'
 	icon_state = "rock_nochance"
-	baseturf = /turf/simulated/floor/plating/asteroid
+	baseturf = /turf/space

```

Comment:
will this have any negative side-effects? like being able to rcd up the asteroid floor?